pkg/cli/gptscript.go

```diff
@@ -64,7 +64,7 @@ type GPTScript struct {
 	Chdir                    string   `usage:"Change current working directory" short:"C"`
 	Daemon                   bool     `usage:"Run tool as a daemon" local:"true" hidden:"true"`
 	Ports                    string   `usage:"The port range to use for ephemeral daemon ports (ex: 11000-12000)" hidden:"true"`
-	CredentialContext        string   `usage:"Context name in which to store credentials" default:"default"`
```

Comment:
Our `pkg` library doesn't support default values for slices, which is why I removed the default tag here.